cudaaligner/src/myers_gpu.cu

```diff
@@ -383,7 +418,8 @@ __global__ void myers_compute_score_matrix_kernel(
 __device__ int32_t myers_backtrace_banded(int8_t* path, device_matrix_view<WordType> const& pv, device_matrix_view<WordType> const& mv, device_matrix_view<int32_t> const& score, int32_t diagonal_begin, int32_t diagonal_end, int32_t band_width, int32_t target_size, int32_t query_size)
 {
     assert(threadIdx.x == 0);
-    using nw_score_t = int32_t;
+    using nw_score_t                    = int32_t;
+    GW_CONSTEXPR nw_score_t out_of_band = numeric_limits<nw_score_t>::max() - 1;
```

Comment:
Why `-1`?

Reply:
Because I do `x + 1 == ...` with the variable I assign it to and need to avoid the integer overflow.

Reply:
see https://github.com/clara-parabricks/GenomeWorks/blob/416d1c7b2721d690f69866b66a4c6b011784b184/cudaaligner/src/myers_gpu.cu#L469

Reply:
Ok. Maybe a comment about it could help, but it's optional

Reply:
true... I added a comment.